spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/route/builder/GatewayFilterSpec.java

```diff
@@ -268,6 +269,8 @@ public GatewayFilterSpec mapRequestHeader(String fromHeader, String toHeader) {
 	 * A filter that can be used to modify the request body. This filter is BETA and may
 	 * be subject to change in a future release.
 	 * @param configConsumer request spec for response modification
+	 * @param <T> the original response body class
```

Comment:
This doesn't look correct. This is the javadoc for the `modifyRequestBody` method.

Reply:
fixed